### ISSUE TITLE
Observability Distributed Tracing - Context Propagation Example

### ISSUE THREAD

**Issue by akakarikos** [dapr/dotnet-sdk#1013]
## Ask your question here

I am having two microservices A and B. There is an external gateway service that calls service A and then service A calls service B. 
`External Gateway > Service A > Service B`

On service A I am using the below code to propagate the context headers and calling the `InvokeMethodAsync()` method.

![image](https://user-images.githubusercontent.com/10879762/214247335-3b76eeb1-0752-49ec-9c3b-4451f9387f14.png)

On service B I am using the below code to extract the context headers.

![image](https://user-images.githubusercontent.com/10879762/214248062-3ef58e97-356c-4412-8f63-5f8e136e6d13.png)

This is not working because service B always creates a new `traceparent` header and ignores the one propagated from service A.

Can you please let me know if this is the correct way to propagate context so the distributed tracing works properly? I cannot find any concrete example in the documentation.

Dapr runtime v1.9.3
Dapr .NET nuget packages v1.9.0

**Comment by msfussell** [comment]
+ @halspang to help answer this.

**Comment by halspang** [comment]
@akakarikos - Have you tried it with no code present for propagating the activities. The SDK has no actual code for propagation and we rely entirely on the functionality already present in AspNet. For my apps, I've never had to add any code and the traces just worked out of the box. 

Does your proposed method enable any specific use-cases? Or do you think you can try and let the SDK/Dapr handle all of that for you?

**Comment by akakarikos** [comment]
Hi, @halspang  thanks for your reply. I removed all the above code and finally, I got it working by just adding the `traceparent` and `tracestate` headers of the existing Activity in the context of the Dapr request message. I had to to the propagation of the headers because my case is this: `Gateway Service -> MicroService A -> MicroService B` and based on the documentation in such cases the header propagation is mandatory.
